### ISSUE TITLE
Messagebox settings remembered even if disabled

### ISSUE THREAD

**Issue by ProgerXP** [ProgerXP/Notepad2e#335]
1. Set Save Settings On Exit to Recent Strings
2. Try to open a large file 
3. In the warning box tick "do not show again"
4. Check the INI (Ctrl+F7) - the choice will be remembered under [Suppressed Messages]`

Is this because Notepad2 immediately remembers the value, not on exit? Like it does with Save History.

**Comment by cshnik** [comment]
You are correct, ``Suppressed Messages`` options are immediately written to INI.
![image](https://user-images.githubusercontent.com/8019354/109341356-e361c980-789c-11eb-89e0-92f2d724ce31.png)
